docs/anomaly-detection/CHANGELOG.md

```diff
@@ -25,6 +25,14 @@ Please find [launch instructions here](/vmanomaly.html#run-vmanomaly-docker-cont
 # tip
 
 
+## v1.8.0
+Released: 2024-01-15
+- FEATURE: Added Univariate [MAD (median absolute deviation)](https://en.wikipedia.org/wiki/Median_absolute_deviation) model support.
```

Comment:
plase provide link to the doc as well

Reply:
@Cambaza, we can just substitute this wiki link with the link to our model section - `/anomaly-detection/components/models.html# ...`, as we already have wiki link there